addons/account/tests/test_account_supplier_invoice.py

```diff
@@ -4,10 +4,26 @@
 class TestAccountSupplierInvoice(AccountingTestCase):
 
     def test_supplier_invoice(self):
-        tax = self.env['account.tax'].create({
-            'name': 'Tax 10.0',
+        tax_fixed = self.env['account.tax'].create({
```

Comment:
I would keep two separate tests. One test for a simple case, another one for more complicated tax combination.